pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1593,6 +1593,11 @@ self: super: {
   # 2022-09-19: https://github.com/haskell/haskell-language-server/issues/3200
   hls-refactor-plugin = dontCheck super.hls-refactor-plugin;
 
+  # 2022-10-27: implicit-hie 0.1.3.0 needs a newer version of Cabal-syntax.
+  implicit-hie = assert self.Cabal-syntax.version == "3.6.0.0"; super.implicit-hie.override {
```

Comment:
This is going to break in 9.4 (and 9.2?) where `Cabal-syntax` is bundled with GHC.

Reply:
Yeah, already noticed and decided that I don‘t care enough about the assertion.